config/kubernetes/wg-k8s-infra/OWNERS

```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - wg-k8s-infra-leads
+  - sig-k8s-infra-leads
```

Comment:
doesn't necessarily need to be in this PR, but do you want to rename the dir to sig-k8s-infra too?